source/extensions/common/wasm/wasm.h

```diff
@@ -296,11 +312,24 @@ class Wasm : public Envoy::Server::Wasm,
 
 private:
   friend class Context;
+  static const uint32_t kMetricTypeMask = 0x3;
```

Comment:
IMHO: all these values and predicates can also declare `constexpr`

Reply:
Changed const->constexpr.  const comes from costexpr for simple types.